src/components/Filters.js

```diff
@@ -10,32 +40,45 @@ class Filters extends React.Component {
       <>
         <label htmlFor="search">
           Search
-          <input id="search" autoFocus placeholder="Search event" />
+          <input
+            id="search"
+            autoFocus
+            placeholder="Search event"
+            onChange={this.onTitleFilterChange}
+          />
         </label>
-        <select>
-          <option>Select city</option>
-          <option>Cracow</option>
-          <option>Wroclaw</option>
-          <option>Warsaw</option>
-          <option>Poznan</option>
-          <option>Gdansk</option>
+        <select onChange={this.onCityFilterChange}>
+          <option value="">Select city</option>
+          <option value="Cracow">Cracow</option>
+          <option value="Wroclaw">Wroclaw</option>
+          <option value="Warsaw">Warsaw</option>
+          <option value="Poznan">Poznan</option>
+          <option value="Gdansk">Gdansk</option>
         </select>
-        <select>
-          <option>Select category</option>
-          <option>Music</option>
-          <option>Arts</option>
-          <option>Business</option>
-          <option>Sport</option>
-          <option>Food</option>
+        <select onChange={this.onCategoryFilterChange}>
+          <option value="">Select category</option>
+          <option value="Music">Music</option>
+          <option value="Arts">Arts</option>
+          <option value="Business">Business</option>
```

Comment:
Please, check if you need to use capital letters for options values in both select tags.